Java/src/main/java/Planes/ExperimentalPlane.java

```diff
@@ -3,14 +3,12 @@
 import models.ClassificationLevel;
 import models.ExperimentalTypes;
 
-public class experimentalPlane extends Plane{
+public class ExperimentalPlane extends Plane{
 
-    private ExperimentalTypes type;
```

Comment:
add getter for this field instead of removing, and use it in equals/hashCode/toString methods